gpii/node_modules/testing/src/CloudBased.js

```diff
@@ -74,6 +75,7 @@ gpii.test.cloudBased.gpiiConfig = function (baseDir) {
 };
 
 gpii.test.cloudBased.buildTestFixture = function (testDef, baseDir) {
+    testDef = $.extend({}, testDef);
```

Comment:
Awesome, Simon!! Thanks so much for finding the source of this issue!

I've added this to my pull request and everything is working (though I infusionized this by using: `testDef = fluid.copy(testDef)` instead of jquery).

Cheers again